app/api/contests/get_administered_list.go

```diff
@@ -0,0 +1,164 @@
+package contests
+
+import (
+	"net/http"
+
+	"github.com/go-chi/render"
+
+	"github.com/France-ioi/AlgoreaBackend/app/service"
+)
+
+type parentTitle struct {
+	// required: true
+	Title string `json:"title"`
+	// required: true
+	LanguageID int64 `json:"language_id,string"`
+}
+
+// swagger:model
+type contestAdminListRow struct {
+	// required: true
+	ItemID int64 `gorm:"column:idItem" json:"id,string"`
+	// Nullable
+	// required: true
+	Title *string `gorm:"column:sTitleTranslation" json:"title"`
+	// Nullable
+	// required: true
+	LanguageID *int64 `gorm:"column:idTitleLanguage" json:"language_id,string"`
+	// required: true
+	TeamOnlyContest bool `gorm:"column:bTeamOnlyContest" json:"team_only_contest"`
+	// required: true
+	Parents []parentTitle `json:"parents"`
+}
+
+// swagger:operation GET /contests/administered contests groups contestAdminList
+// ---
+// summary: Get the contests that the user has administration rights on
+// description: >
+//                For all items that are timed contests and for that the user is a contest admin
+//                (has `solutions` or `full` access), returns item info (`id`, `title`, `team_only_contest`, parents' `title`-s).
+//                Only parents visible by the user (`full`, `partial`, `gray`) are listed.
+//
+//
+//                Each title is returned in the user's default language if exists,
+//                otherwise the item's default language is used. If there are no titles for these languages,
+//                then the title gets skipped (i.e. omitted in the parents array (for a parent) or
+//                having `title` = null (for the item itself)).
```

Comment:
Actually, I do not find this consistent. In the first list you put them as null, in the second one you omit the entry...
We have to choose between the two options. I would go in favour of omitting entries when there is no title entry (which should not happen anyway).

Reply:
Hmm... As it should not happen anyway, I would prefer to keep them

Reply:
fixed